src/managers/GuildChannelManager.js

```diff
@@ -59,21 +59,26 @@ class GuildChannelManager extends BaseManager {
    */
 
   /**
-   * Creates a new channel in the guild.
-   * @param {string} name The name of the new channel
-   * @param {Object} [options] Options
-   * @param {string} [options.type='text'] The type of the new channel, either `text`, `voice`, `category`, `news`,
+   * Options used to create a new channel in a guild.
+   * @typedef {Object} GuildCreateChannelOptions
+   * @property {string} [type='text'] The type of the new channel, either `text`, `voice`, `category`, `news`,
    * `store`, or `stage`
-   * @param {string} [options.topic] The topic for the new channel
-   * @param {boolean} [options.nsfw] Whether the new channel is nsfw
-   * @param {number} [options.bitrate] Bitrate of the new channel in bits (only voice)
-   * @param {number} [options.userLimit] Maximum amount of users allowed in the new channel (only voice)
-   * @param {ChannelResolvable} [options.parent] Parent of the new channel
-   * @param {OverwriteResolvable[]|Collection<Snowflake, OverwriteResolvable>} [options.permissionOverwrites]
+   * @property {string} [topic] The topic for the new channel
+   * @property {boolean} [nsfw] Whether the new channel is nsfw
+   * @property {number} [bitrate] Bitrate of the new channel in bits (only voice)
+   * @property {number} [userLimit] Maximum amount of users allowed in the new channel (only voice)
+   * @property {ChannelResolvable} [parent] Parent of the new channel
+   * @property {OverwriteResolvable[]|Collection<Snowflake, OverwriteResolvable>} [permissionOverwrites]
    * Permission overwrites of the new channel
-   * @param {number} [options.position] Position of the new channel
-   * @param {number} [options.rateLimitPerUser] The ratelimit per user for the channel
-   * @param {string} [options.reason] Reason for creating the channel
+   * @property {number} [position] Position of the new channel
+   * @property {number} [rateLimitPerUser] The ratelimit per user for the new channel
+   * @property {string} [reason] Reason for creating the new channel
+   */
+
+  /**
+   * Creates a new channel in the guild.
+   * @param {string} name The name of the new channel
+   * @param {GuildCreateChannelOptions} [options={}] Options for creating the new channel
```

Comment:
I think having [options = {}] is redundant because the [] indicates that the parameter is optional and saying that an empty object is assumed is already implicit when the parameter is omitted. I know this is done many times across the lib but I still think it's worth the change (maybe change others too, would need feedback from the maintainers)

Reply:
Yes, the `[]` states that the param is optional. Doing `[options = {}]` shows `{}` under default in the docs which will not be there if we remove the `= {}` part. I think this is better because it distinct itself from optional param that are not objects.

Reply:
Yeah I know how that's displayed but if the options parameter is already marked as an object (by the typedef), adding {} wouldn't add much information imo